src/app/work-item/work-item.service.ts

```diff
@@ -49,8 +49,10 @@ export class WorkItemService {
       return this.http
       .get(this.workItemTypeUrl)
       .toPromise()
-      .then((response) => {
-        this.workItemTypes = process.env.ENV != 'inmemory' ? response.json() as WorkItemType[] : response.json().data as WorkItemType[];
+      // .then((response) => {
+       // this.workItemTypes = process.env.ENV != 'inmemory' ? response.json() as WorkItemType[] : response.json().data as WorkItemType[];
+         .then(response => {
+        this.workItemTypes = response.json().data as WorkItemType[];
```

Comment:
@ldimaggi did you change this because you got the tests work without the inmemory?